pkg/server/handler/schema.go

```diff
@@ -51,8 +51,8 @@ type schemaField struct {
 	TypeName string `mapstructure:"type" json:"type"`
 }
 
-func (resp *schemaResponse) Encode(data map[string]skydb.RecordSchema) {
-	resp.Schemas = make(map[string]schemaFieldList)
+func encodeRecordSchemas(data map[string]skydb.RecordSchema) map[string]schemaFieldList {
```

Comment:
Move this to recordutil or (new) schemautil.

Basically we try not to have different handler files (record and schema) with code calling code in other file.